### PR TITLE
Update Dockerfile updated

### PR DESCRIPTION
**Pull request by olamyde**
Update Dockerfile updated

### CODE REVIEW

**Comment by EDEMOS80** [comment]
good job

File: Dockerfile [hunk 1]
```diff
@@ -1,2 +1,29 @@
-FROM httpd:2.4
-COPY ./public-html/ /usr/local/apache2/htdocs/
+FROM some-base-image
+
```
Comment (devopseasylearning) [inline Dockerfile:2]: PLease remove empty  line

File: Dockerfile [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+ENV section=V1
+ENV PORT=5278
+ENV HOST=devopseasylearning.com
+ENV zone=prod
+
+# Company information labels
+LABEL company.phone="+123456789"
+LABEL company.name="DEVINE Corporation"
+LABEL company.address="123 Main Street, Cityville, State, Zip"
+
```
Comment (devopseasylearning) [inline Dockerfile:14]: PLease remove empty  line

File: Dockerfile [hunk 3]
```diff
@@ -1,2 +1,29 @@
...
+# Set the working directory
+WORKDIR /app
+
+# Copy the PostgreSQL configuration file
+COPY postgres.conf /var/lib/postgres.conf
+
```
Comment (devopseasylearning) [inline Dockerfile:20]: PLease remove empty  line

File: Dockerfile [hunk 4]
```diff
@@ -1,2 +1,29 @@
...
+# Set PostgreSQL environment variables
+ENV DB_HOST=localhost
+ENV DB_PORT=5432
+ENV DB_NAME=mydatabase
+ENV DB_USER=myuser
+ENV DB_PASSWORD=mypassword
+
```
Comment (devopseasylearning) [inline Dockerfile:27]: PLease remove empty  line

File: Dockerfile [hunk 5]
```diff
@@ -1,2 +1,29 @@
...
+ENV app=DEVINE
+ENV section=V1
+ENV PORT=5278
+ENV HOST=devopseasylearning.com
+ENV zone=prod
+
+# Company information labels
+LABEL company.phone="+123456789"
+LABEL company.name="DEVINE Corporation"
+LABEL company.address="123 Main Street, Cityville, State, Zip"
```
Comment (devopseasylearning) [inline Dockerfile:13]: break this down

**Pull request closed without merging**